nautobot/cloud/tests/test_filters.py

```diff
@@ -75,16 +77,26 @@ class CloudNetworkPrefixAssignmentTestCase(FilterTestCases.FilterTestCase):
     ]
 
 
+class CloudServiceNetworkAssignmentTestCase(FilterTestCases.FilterTestCase):
+    queryset = models.CloudServiceNetworkAssignment.objects.all()
+    filterset = filters.CloudServiceNetworkAssignmentFilterSet
+    generic_filter_tests = [
+        ("cloud_network", "cloud_network__id"),
+        ("cloud_network", "cloud_network__name"),
+        ("cloud_service", "cloud_service__id"),
+        ("cloud_service", "cloud_service__name"),
+    ]
+
+
 class CloudServiceTestCase(FilterTestCases.NameOnlyFilterTestCase):
     queryset = models.CloudService.objects.all()
     filterset = filters.CloudServiceFilterSet
     generic_filter_tests = [
         ("cloud_account", "cloud_account__id"),
         ("cloud_account", "cloud_account__name"),
-        ("cloud_network", "cloud_network__id"),
-        ("cloud_network", "cloud_network__name"),
+        ("cloud_networks", "cloud_networks__id"),
+        ("cloud_networks", "cloud_networks__name"),
         ("cloud_resource_type", "cloud_resource_type__id"),
         ("cloud_resource_type", "cloud_resource_type__name"),
-        ("description",),
```

Comment:
Why removed?

Reply:
This must have been a bad merge resolution. I'll add it back